docs/source/settings.rst

```diff
@@ -234,7 +234,13 @@ keepalive
 
 The number of seconds to wait for requests on a Keep-Alive connection.
 
-Generally set in the 1-5 seconds range.
+Generally set in the 1-5 seconds range for servers with direct connection
+to the client (=when you don't have separate load balancer). When
```

Comment:
Or 'e.g.', since it might be something other than a load balancer (like an app server that has a connection to another backend service that uses gunicorn).